workflows/cascading-workflows.md

```diff
@@ -0,0 +1,15 @@
+# Cascading Workflows
+
+This guide will walk you through how to deploy a workflow that runs after another workflow. If a source workflow is provided, the target workflow will be triggered at the end of each successful run of the source workflow. Each workflow can have at most one source workflow. Aqueduct will prevent any cycles from forming amongst cascading workflows.
```

Comment:
Clarify that if the source workflow does not complete successfully, then the target workflow won't be triggered.

Reply:
How will we prevent the cycles? What will the user see?